.eslintignore

```diff
@@ -0,0 +1 @@
+test/async-function-tests.js
```

Comment:
@rwjblue Thanks. We can also use ESLint's `no-restricted-syntax` to disallow async functions if we don't want to use them anywhere else in the code. (I think async functions and await expressions are the only new syntax in ES2017 unless my memory is failing me.)